### ISSUE TITLE
.spam setup doesn't prevent command from being executed again while in setup

### ISSUE THREAD

**Issue by MightyKnight** [ImpactDevelopment/ImpactClient#633]
### What happens?
Multiple spammer setups can be started with the `.spam setup` command even while already setting up one.
This results in multiple and confusing messages unnecassary trash creation in `spam.json` and in a worst-case scenario it also overrides pre-made configs in `spam.json`.

<details> 
  <summary><strong>Messed up spam.json after bug

</strong></summary>
 
```
".spam setup":{
  "delay":1000,
  "messages":[
    "1000",".spam setup","Hello","this","is","annoying"
  ]
},
"1000":{
  "delay":1000,
  "messages":[
    ".spam setup","Hello","this","is","annoying"
  ]
}

```

</details>

![2018-07-02_22 31 54](https://user-images.githubusercontent.com/38622942/42185082-f82f5a50-7e47-11e8-93fc-4593e79561f2.png)

### Exact steps to reproduce?

- Use `.spam setup` two or more times in a row

<details> 
  <summary><strong>Additional Client Info</strong></summary>
 
- Impact Version: 4.2 for 1.12.2
- Using Optifine? No
- Operating System: Windows 10 64-bit
</details>

**Comment by ZeroMemes** [comment]
4.3 released, bumping to 4.4.

**Comment by ZeroMemes** [comment]
This should be done by allowing events to be **stoppable**, preventing them from being passed to the succeeding listeners.

**Comment by ZeroMemes** [comment]
Fixed in 4.4
